openapi/components/schemas/KycDocument/IdentityMatches.yaml

```
@@ -101,6 +101,10 @@ properties:
     description: Specifies if an identity document has matching face proof.
     type: boolean
     example: false
+  isTampered:
+    description: Specifies if an identity document may have been tampered with.
```

nitpick: consistency with the other description (last nitpick!)
```suggestion
    description: Specifies if an identity document has been tampered with.
```